packages/webview_flutter/webview_flutter_wkwebview/pubspec.yaml

```diff
@@ -0,0 +1,29 @@
+name: webview_flutter_wkwebview
+description: A Flutter plugin that provides a WebView widget based on Apple's WKWebView control.
+repository: https://github.com/flutter/plugins/tree/master/packages/webview_flutter/webview_flutter_wkwebview
+issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3A%22p%3A+webview%22
+version: 0.0.1
+
+environment:
+  sdk: ">=2.12.0 <3.0.0"
+  flutter: ">=2.0.0"
```

Comment:
To avoid a follow-up PR: this can be 2.14 and 2.5 respectively, since we aren't supporting iOS 8 now. We're in the process of updating plugins accordingly.

Reply:
Done.